tests/net/context/src/main.c

```diff
@@ -1160,98 +987,67 @@ NET_DEVICE_INIT(net_context_test, "net_context_test",
 		&net_context_if_api, _ETH_L2_LAYER,
 		_ETH_L2_CTX_TYPE, 127);
 
-static bool test_init(void)
+static void test_init(void)
 {
 	struct net_if_addr *ifaddr;
 	struct net_if_mcast_addr *maddr;
 	struct net_if *iface = net_if_get_default();
 
-	if (!iface) {
-		TC_ERROR("Interface is NULL\n");
-		return false;
-	}
+	zassert_not_null(iface, "Interface is NULL");
 
 	ifaddr = net_if_ipv6_addr_add(iface, &in6addr_my,
 				      NET_ADDR_MANUAL, 0);
-	if (!ifaddr) {
-		TC_ERROR("Cannot add IPv6 address %s\n",
-			 net_sprint_ipv6_addr(&in6addr_my));
-		return false;
-	}
+	zassert_not_null(ifaddr,
+				"Cannot add IPv6 address ");
 
 	ifaddr = net_if_ipv4_addr_add(iface, &in4addr_my,
 				      NET_ADDR_MANUAL, 0);
-	if (!ifaddr) {
-		TC_ERROR("Cannot add IPv4 address %s\n",
-			 net_sprint_ipv4_addr(&in4addr_my));
-		return false;
-	}
+	zassert_not_null(ifaddr,
```

Comment:
same here

Reply:
@jukkar Done

Reply:
@jukkar  Can  you please review so that I can merge in v1.10 ?